### PR TITLE
Make a separate dir to store covreport files 

### PR DESCRIPTION
**Pull request by Navidem**
This can prevent the side effect reported here  #7188

### CODE REVIEW

File: infra/base-images/base-runner/coverage [hunk 1]
```diff
@@ -31,11 +31,12 @@ COVERAGE_OUTPUT_DIR=${COVERAGE_OUTPUT_DIR:-$OUT}
 
 DUMPS_DIR="$COVERAGE_OUTPUT_DIR/dumps"
 FUZZER_STATS_DIR="$COVERAGE_OUTPUT_DIR/fuzzer_stats"
+COVERAGE_REPORT_DIR="$COVERAGE_OUTPUT_DIR/covreports"
```
Comment (jonathanmetzman) [inline infra/base-images/base-runner/coverage:34]: I have an annoying idea. Maybe we should call these reports textcov to distinguish them from the HTML reports.
I think that's what they are right?
If you don't want to change the .covreport suffix, maybe just change the directory name (and variable name).
WDYT?

Comment (Navidem) [reply infra/base-images/base-runner/coverage]: well, yes they are the output of llvm-cov show. Made changes.

**Review by jonathanmetzman — APPROVED** [review]
lgtm, thanks for the fix!

**Comment by jonathanmetzman** [comment]
Note I think the build failures in CI are because of a "bug" in our CI system. We diff against master's HEAD instead of the base commit of the branch to figure out if any fuzzers have been modified. You can avoid these by pulling from master into your branch.

**Pull request merged**